Code/GraphMol/Chirality.cpp

```diff
@@ -1089,6 +1088,88 @@ void buildCIPInvariants(const ROMol &mol, DOUBLE_VECT &res) {
   }
 }
 
+//! Lightweight sortable wrapper that references a CIP entry and keeps track of
+//! the current rank.
+struct SortableCIPReference {
+  SortableCIPReference(CIP_ENTRY* cipRef, const int atomIdx): cip(cipRef), atomIdx(atomIdx) {}
```

Comment:
It wouldn't be terrible to confirm here that `cipRef` is not nullptr
It's not directly relevant right now, but if this code were ever to be used elsewhere it could become important

Reply:
Done